cli/utilities/containers.py

```diff
@@ -194,4 +195,20 @@ def ps(self, all=None, filter=None):
         if filter:
             for key, val in filter.items():
                 cmd += f" --filter {key}={val}"
+
+        if format:
+            cmd += f" --format {format}"
+
+        return self.execute(sudo=True, cmd=cmd)
+
+    def exec(self, options, container_id, command):
+        """
+        Return output of command in a running container
+        Args:
+            all (bool): Show all containers
+            filter (dict): Key value to filter
+            format (string): Format values
+        """
+        cmd = f"{self.base_cmd} exec -{options} {container_id} {command}"
```

Comment:
use exact parameters mentioned for exec. Refer - https://docs.podman.io/en/latest/markdown/podman-exec.1.html

Reply:
@vamahaja 
Updated